addons/binding/org.openhab.binding.tankerkoenig/README.md

```diff
@@ -116,10 +117,15 @@ Users should check the log for any reports to solve the reason for this status.
 next to the OFFLINE not return the status "OK", which could for an example be caused by a banned API-key. In such a case the polling of price-data is stopped. 
 If no valid response is received the polling will continue. On the next receipt of a valid message Webservice and Station(s) will go ONLINE again. 
 
+-How to set the switch item for the channel holiday?
+
+The correct usage of opening times needs the information if the actual day is a holiday. The binding expects a switch item linked to the Webservice channel holiday.
+This switch can be set either manually (only suggested for testing!), by a rule [openHAB1-addons rules] or by the usage of the CALDAV binding with a calendar.
+
 ## Tankerkönig API
 
 *  https://creativecommons.tankerkoenig.de/  (sorry, only available in german)
 
    [MTS-K]: <https://www.bundeskartellamt.de/DE/Wirtschaftsbereiche/Mineral%C3%B6l/MTS-Kraftstoffe/Verbraucher/verbraucher_node.html>
-
+   [openhab1-addons rules]: <https://github.com/openhab/openhab1-addons/wiki/Samples-Rules#how-to-calculate-public-holidays>
```

Comment:
Does this remove the existing holiday code in this binding or was that already removed?